src/url-params.ts

```diff
@@ -109,6 +146,17 @@ export class AnalyticsUrlParams {
 		return `${URL_PARAM_DEVICE_ID}=${encodeURIComponent(ids.join(','))}`;
 	}
 
+	/**
+	 * @return part of the session ID query parameter string that can be appended to URLs
+	 */
+	getSessionIdQueryString(): string {
```

Comment:
What do you think defining an additional method here to consolidate the results of `getDeviceIdsQueryString` and `getSessionIdQueryString`?
Mainly for convenience reasons. So the component maintainer could retrieve everything in one go to append to the URL.

Reply:
I am adding it as an additional method `getQueryString`.
As I understand, there might be legitimate cases where we only pass device or session IDs so leaving those methods as they are seems wise (besides the obvious backwards-compatibility).